### PR TITLE
Added license badge in README.md file

### PR DESCRIPTION
**Pull request by sumitkharche**

### CODE REVIEW

**Comment by sumitkharche** [comment]
@eddiewebb Can you please review changes? Its really simple change. We are good to merge this PR.

**Comment by sumitkharche** [comment]
@eddiewebb Did you get chance to look into this PR?

**Comment by eddiewebb** [comment]
Hi @sumitkharche, thanks for the contribution.

I suspect that you're here because of Hacktoberfest, and am so grateful, and happy to see folks getting involved.  

However, I need to tell you:

1. project maintainers are busy, it's not fair to expect them to review and approve PRs in a couple of days.  Please be patient.
2. It's good to ask if changes are desired before making them.  For example - All orbs are licensed to community under MIT, so I am not sure how important a license badge is.  But an orb version badge would be very cool.   -- Opening an issue first or the PR as a suggestion for feedback is a great way to have this kind of dialog rather than saying "Its really simple change. We are good to merge this PR."


OK, so for this specific PR, i'll add comments in a review, but I would like to see some changes.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # CircleCI Concurrency Control Orb
 
+[![CircleCI](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master.svg?style=svg)](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master) [![GitHub license](https://img.shields.io/github/license/eddiewebb/circleci-queue)](https://github.com/eddiewebb/circleci-queue/blob/master/LICENSE)
```
Comment (eddiewebb) [inline README.md]: License is fine.   Markdown will allow the images on their own lines, but show inline. This presents the same, but much easier to read source.

https://jbt.github.io/markdown-editor/#pY4xC8IwEIX3/oqKix3M7YI41KIF6eJYHEx6JAdJo83F4r9Xa1GXTm7Hu/c+vnlaYZ9uvYoOW06Selbn1CmLeXlaGOZLWAGoIVEklHegDWDTEPYo5eezvEaMCNwhgjsHxk6Em94EvltcP6/sP1j28toR76NMLSlsA36B5LQIhtA2QZAHTWyihLE2if8xei8Gn0kZab0cZeBQ5kV1LLIH

Comment (eddiewebb) [inline README.md]: Change circleci status badge to use sheilds.io as well for consistent style

File: README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+
```
Comment (eddiewebb) [inline README.md:6]: Please add orb source shield as well! That takes format of

```
[![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/PROJECT/NAME)]
```

**Review by eddiewebb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sumitkharche) [reply README.md]: Done with changes

Comment (sumitkharche) [reply README.md]: Changed circleci status badge

**Review by eddiewebb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sumitkharche) [reply README.md:6]: Added CircleCI Orb Version badge

**Pull request merged**